### DISCUSSION TITLE
Support multiple themes

### DISCUSSION THREAD

**Discussion by sakanjo** [spatie/shiki-php#22] [Ideas]
It would be awesome to implement multiple themes, especially for light/dark mode sites,

Ref: https://shiki.style/guide/dual-themes.

**Comment by Barbapapazes** [comment]
I agree with this. Having the support for both light and dark themes is mandatory for website with a ligtht and a dark mode.

**Reply by Barbapapazes** [reply]
I have open a PR: https://github.com/spatie/shiki-php/pull/26

**Reply by Barbapapazes** [reply]
This is merged!

**Reply by sakanjo** [reply]
Many thanks.
